src/components/molecules/MenuModal/styled.js

```diff
@@ -18,7 +18,7 @@ export const Container = styled.TouchableOpacity`
   background-color: ${Color.background};
   width: 230px;
   height: 100px;
-  margin-top: 18.5%;
+  margin-top: 14%; /* // ADRI MUEVE ESTE % PARA QUE QUEDE CHIDO EN IOS EL MODAL */
```

Comment:
Remember, you shouldn't leave directions to your peers on what you push to GitHub.